pkgs/by-name/fi/filius/package.nix

```
@@ -0,0 +1,125 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchzip,
+  fetchurl,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+  jre,
+}:
+let
+  # we cache potentially unstable upstream inputs (icon and .zip file) via https://web.archive.org - this is a standard procedure in Nixpkgs
+  icon128 = fetchurl {
+    url = "https://web.archive.org/web/20240623123147/https://dl.flathub.org/repo/appstream/x86_64/icons/128x128/de.lernsoftware_filius.Filius.png";
+    sha256 = "sha256-xES6RWf2KiNsukD6GwTuPyiDI6SseADGLb1AkBSTgd8=";
+  };
+  icon32 = fetchurl {
+    url = "https://gitlab.com/filius1/filius/-/raw/09fa7d5bdc60d5b4a2ab0e24311d54516a400913/src/deb/filius32.png";
+    sha256 = "sha256-sVsViRALhiVJFSxfVb9K6Q4M7Y2UEvZh6oOB9BQumh8=";
+  };
+  mimetype_xml = fetchurl {
+    url = "https://gitlab.com/filius1/filius/-/blob/09fa7d5bdc60d5b4a2ab0e24311d54516a400913/src/deb/application-filius-project.xml";
+    sha256 = "sha256-EyLoHhshTQYgVDOWPokpDH0AapX3aqtb+fAiWv2G51I=";
+  };
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "filius";
+
+  # UPDATE instructions
+  #
+  # - Open https://www.lernsoftware-filius.de/Herunterladen and copy download link to clipboard.
+  #   (e.g. https://www.lernsoftware-filius.de/downloads/Setup/filius-2.5.1.zip)
+  # - identify the new version string.
+  version = "2.5.1";
+
+  # - Open http://web.archive.org and paste download link from clipboard into "Save Page Now" field and hit the "Save Page" button.
+  # - Unselect "Save Error Pages" and hit "Save Page" again.
+  # - Wait for the archive link to be generated and copy it to the url field - adjust hash accordingly.
+  src = fetchzip {
+    url = "https://web.archive.org/web/20240623122638/https://www.lernsoftware-filius.de/downloads/Setup/filius-2.5.1.zip";
+    hash = "sha256-UrtmL0jHf86vXXCtRJoaXAyElBvUUF1jGpBcO0GV6Lg=";
+    stripRoot = false;
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    copyDesktopItems
+  ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      type = "Application";
+      name = "Filius";
+      desktopName = "Filius";
+      genericName = "Computer Network Simulator";
+      comment = "A Computer Network Simulator for Secondary Schools";
+      icon = "filius";
+      exec = "filius";
+      terminal = false;
+      mimeTypes = [ "application/filius-project" ];
+      categories = [ "Education" ];
+      startupNotify = false;
+      extraConfig = {
+        "GenericName[en]" = "Computer Network simulator";
+        "GenericName[de]" = "Computer-Netzwerksimulator";
+        "Comment[en]" = "A Computer Network Simulator for Secondary Schools";
```

```suggestion
        "Comment[en]" = "A computer network simulator for secondary schools";
```
Same issue with secondary schools.